commit dab22a155b78731803058c6a746396de03220362
Author: Martin Cermak <mcermak@redhat.com>
Date:   Tue Nov 25 15:58:29 2014 +0100

    RHBZ1167652: Boot time probing feature fix
    
    * initscript/systemtap.in: Fix regenerating initramfs image
      without the stap module

diff --git a/initscript/systemtap.in b/initscript/systemtap.in
index 9443384..cd65b3f 100755
--- a/initscript/systemtap.in
+++ b/initscript/systemtap.in
@@ -906,8 +906,13 @@ onboot () {
     do_failure "Failed to make temporary file in $dir"
     return 1
   fi
-  # Create the initramfs image with stap module enabled.
-  out=$($DRACUT --add stap --force $TMPINITRAMFS $KRELEASE 2>&1)
+  if [ ! "$ss" ]; then
+    # Create the initramfs image without stap module enabled.
+    out=$($DRACUT --force $TMPINITRAMFS $KRELEASE 2>&1)
+  else
+    # Create the initramfs image with stap module enabled.
+    out=$($DRACUT --add stap --force $TMPINITRAMFS $KRELEASE 2>&1)
+  fi
   # dracut will report success even if some modules (e.g. stap) failed
   # to install some files, so we need to be a bit more involved in
   # checking for errors
